app/src/main/res/layout/lessons_not_started_chapter_view.xml

```diff
@@ -48,7 +48,7 @@
       android:layout_height="0dp"
       android:fontFamily="sans-serif"
       android:gravity="center|start"
-      android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
+      android:importantForAccessibility="yes"
```

Comment:
From yours and Rajat's videos, the in progress chapter name is still being read twice. Have you checked to see if that is the case if this is set to no?

Reply:
Yes, please try that out

Reply:
Okay Thanks @adhiamboperes.

Reply:
Done.